stdlib/Future/docs/src/index.md

```diff
@@ -0,0 +1,17 @@
+# Future
+
+The `Future` module implements future behavior of already existing functions,
+which will replace the current version in a future release of Julia.
+
+```@meta
+DocTestSetup = :(using Future)
+```
+
+```@docs
+Future.copy!
```

Comment:
Should we really document this function now that it's planned to be deprecated?

Reply:
Where are those plans?

Reply:
In the docstring of this function. Since julia 1.1, the functionality is availble in `Base`. But if this PR ends up updating the doc for julia 1.0, it may make sense to document it here anyway.

Reply:
Right. We have one docs for Julia 1 so I think it makes sense to document it, and we don't know when we get the chance to actually deprecate this function.